metricbeat/module/system/cpu/cpu.go

```diff
@@ -100,6 +100,11 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 		"5":  loadStat.Load5,
 		"15": loadStat.Load15,
 	}
+	cpuStat["load_normalized"] = common.MapStr{
```

Comment:
Should we go here with `load.1` and `load.normlized.1` or `load.1.value` and `load.1.normlized`. I think at least both should be in the namespace `load`. Not sure about the naming below it.